lib/awesome_spawn.rb

```diff
@@ -101,6 +101,29 @@ def run!(command, options = {})
     command_result
   end
 
+  # Execute `command` in a detached manner
+  #   by default the :err, and :out are redirected to `/dev/null`
+  #   by default :pggroup is true (to make a daemon process)
+  #   by default :pggroup_new is true (needed for windows)
```

Comment:
I love this, but there are times you want to run a detached child process...perhaps `run_detached` stays as you have here (which I like) and maybe we can add a `run_child` or `run_detached_child` which does the same thing, but keeps the pgroup the same.

Reply:
Think this can be done w/ the current implementation of spawn.

```
  process group:
    :pgroup => true or 0 : make a new process group
    :pgroup => pgid      : join the specified process group
    :pgroup => nil       : don't change the process group (default)
  create new process group: Windows only
    :new_pgroup => true  : the new process is the root process of a new process group
    :new_pgroup => false : don't create a new process group (default)
```